src/components/common/SubmissionViewer.js

```diff
@@ -75,4 +70,37 @@ const SubmissionViewer = ({
   );
 };
 
-export default SubmissionViewer;
+export const SubmissionModal = ({ isModalVisible, onClose, src }) => {
+  return (
+    <div
+      className={`submission-modal-wrapper center-content-flex popup-animated ${
+        isModalVisible && 'showing'
+      }`}
+    >
+      <div className="submission-modal-wrapper__top-bar">
+        <button className="close-btn" onClick={onClose}>
+          Close
+          <CloseOutlined />
+        </button>
+      </div>
+      <div className="submission-modal-wrapper__content center-content-flex">
+        <img src={src} alt="submission" />
+      </div>
+      <div className="submission-modal-wrapper__bottom-bar flex--column center-content-flex">
+        <div>
+          <h4 className="atma">How do you feel about this story?</h4>
```

Comment:
I think h2 or h3 would be the correct header tags here because of the hierarchy, but I might be wrong. I can look more at it later.

Reply:
Actually looking at it again, I don't think this should be hard-coded into the SubmissionModal component because other pages outside of voting won't require emoji feedback, they will just allow the user to view the submission.

Reply:
@drew-ross That's what I was initially thinking too.  I initially had a boolean that conditionally rendered this part, but I took it out bc of the following thought process I had:
- This component is only used here (voting) and on the one meet team/view your squad portfolio page
- There are no alternate figma designs for this modal, and I think the discussion about it being a full-page modal was centered around because it had the emoji feedback which was another modal inside of it
- I thought maybe (as a result), the meet your team would either have a different modal or would have access to adding emojis.

It would be super easy to just put a boolean in for displaying the emoji feedback part or not, but those ^ made me think this modal might just always have the emoji part

What do you think?

Reply:
I'll swap out the `h4` tags.  For hierarchy, I think you're right.  Should this be a `h` tag in the first place, or should it be something else? I'm not really sure if it's a header

Reply:
Sorry if the Figma is misleading. This modal is actually used on all of the SubmissionViewers (all the circular submission previews with the with the zoom icon). So anywhere there's a SubmissionViewer, the component can be clicked to open the modal. We should probably allow for optional children components where we would put any extra content, such as the "How do you feel about this story" text and the EmojiPicker.